src/resources/order/orders.controller.js

```diff
@@ -108,3 +112,32 @@ export const cancelOrder = async (req, res) => {
     );
   }
 };
+
+export const getAllOrders = async (req, res) => {
+  try {
+    let { page, pageSize } = req.query;
+    page = parseInt(page, 10);
```

Comment:
- Let's give a fallback option for page and pageSize, this is important because visiting the first page for the first time doesn't require a page number. this line should read `page = parseInt(page, 10) || 1;` This way, we always default to page 1 if none is provided

- Thinking about it, the name 'pageSize' should be 'limit'.

- Do we really need to pass a Limit in the param? since we don't expect it to change we can just fix it in the code

Reply:
The frontend view has a box where you can choose the limit you want and it's in even numbers, 12, 16, 20, something like that. that can be removed and page size will be given a default.

page and pageSize is what objection.js use for offset and limit chain, so I just used the same for the queries too.

Reply:
Oh okay